drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -50,6 +50,26 @@ class DragAndDropBlock(XBlock):
         default=1
     )
 
+    item_background_color = String(
+        display_name="Item background color",
+        help=(
+            "Background color to use for draggable items. "
+            "Defaults to color specified by current theme, or blue if no theme is set."
+        ),
+        scope=Scope.settings,
+        default=""
```

Comment:
@bradenmacdonald OK, I added trailing commas to the fields that this PR introduces, and also took the opportunity to add trailing commas to all of the remaining fields.